### PR TITLE
Add timezone to datetime.utcnow() in current_login

### PR DESCRIPTION
**Pull request by avilaton**
Proposal for closing #466. Please @jadkik check this out.

### CODE REVIEW

**Comment by avilaton** [comment]
How naive was I, this PR fails... The current sqlalchemy tests do not use timezone so comparing them fails. Maybe it has to be made into an config option to use timezones, @mattupstate probably has a better idea.

**Comment by jonafato** [comment]
Thanks for the PR! It seems like there's demand for this feature, so it's definitely worth considering. A few notes:

1. The solution should be configurable and default to the current behavior to preserve backward compatibility.

2. `pytz` is a bit of a heavy dependency to add just for this purpose. Python 3.2+ includes [`datetime.timezone.utc`](https://docs.python.org/3/library/datetime.html#datetime.timezone.utc). The Python 2 [`tzinfo` documentation](https://docs.python.org/2/library/datetime.html#tzinfo-objects) includes an example UTC tzinfo class definition. It seems like using these (possibly in the 2-to-3 compatibility sections of `utils.py`) would be sufficient to fix the issue.

3. If a third party solution is used, its requirement should not be pinned to a specific version. With the case of `pytz` (and any other library that tracks timezone information), you likely always want the most up-to-date version.

4. `pytz` includes all (500+) of the Olson timezone files in its distribution. This is unfortunately a problem on Google AppEngine (which has a very small file limit), leading to libraries like [gaerpytz](https://gaerpytz.appspot.com/) as workarounds. Adding pytz as a Flask-Security dependency would break applications being deployed on GAE, which is undesirable.

TL;DR: the solution here is probably configurable behavior to use a UTC `tzinfo` implementation taken directly from the Python docs rather than depending on another library.

**Comment by avilaton** [comment]
Thanks to you @jonafato for the explanation. I guess I would have to write a test suite that uses something other than sqlite to be able to test the feature. I will try to figure out how to do that.

**Comment by erwagasore** [comment]
Is it possible to make this `new_current_login_at` configurable similarly to `SECURITY_TRACKABLE` and let user choose what to do being naive or aware

```python
SECURITY_TRACKABLE = True
# SECURITY_DATE_AWARENESS = datetime.utcnow()
SECURITY_DATE_AWARENESS = arrow.utcnow()
```

This will arrow users to choose which library to pip install

**Comment by philsalesses** [comment]
+1 This is a critical bug imho
